build-scripts/hack/update-coredns-chart.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+VERSION="1.36.0"
+DIR=$(realpath $(dirname "${0}"))
+
+CHARTS_PATH="$DIR/../../k8s/manifests/charts"
+
+cd "$CHARTS_PATH"
```

Comment:
Maybe we can use `-d` flag in `helm pull` instead of `cd`?

Reply:
We could but that would mean changing the other scripts to do that as well. The user's directory is not changed when he runs the script, so functionally it's the same thing.